### PR TITLE
Openshift build and deploy scripts

### PR DESCRIPTION
**Pull request by slemrmartin**
This PR contains scripts which can deploy mock source and other topological_inventory/sources services (actually for namespace test-mslemr).

User needs to be logged through `oc` command.

It also contains curl helpers like `api-get`, `api-post`, `sources-api-get` etc.

See #18 for documentation (copy of README)

### CODE REVIEW

**Comment by slemrmartin** [comment]
Note: after merge of #23, config maps have to be changed!

**Comment by slemrmartin** [comment]
@Ladas pls review

File: scripts/functions/common.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+         --silent \
+         "${TOPOLOGICAL_INVENTORY_API_SERVICE_HOST}:${TOPOLOGICAL_INVENTORY_API_SERVICE_PORT}/${BASE_PATH}/$1"
+}
+
+# Usage: api_delete <path>
+# Example: api_delete "sources/1"
+function api_delete {
+    curl --request DELETE \
+         --header "x-rh-identity: ${X_RH_IDENTITY}" \
+         --header "x-rh-insights-request-id: 1" \
```
Comment (Ladas) [inline scripts/functions/common.sh:36]: what is this header for?

**Review by Ladas — APPROVED** [review]
:+1: awesome work

going forward, we need to figure out how to leverage the e2e build templates, so we don't duplicate them here

File: scripts/functions/common.sh (hunk 1 above)
Comment (slemrmartin) [reply scripts/functions/common.sh:36]: hm, it was "new feature" in api-common, which requested this header. But it was intended for requests using 3-scale. It's fixed and not needed now for internal services I think

**Pull request merged**